content/en/security/cloud_security_management/setup/csm_cloud_workload_security/agent/kubernetes.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Enabling CSM Cloud Workload Security on Kubernetes
+kind: documentation
+code_lang: kubernetes
+type: multi-code-lang
+code_lang_weight: 60 # a number that represents relative weight. 
+---
+
+Use the following instructions to enable [CSM Threats][1] on Kubernetes. To learn more about the supported deployment types for each CSM feature, see [Setting Up Cloud Security Management][2].
+
+{{< tabs >}}
+
+{{% tab "Datadog Operator" %}}
+
+1. Add the following to the `spec` section of the `datadog-agent.yaml` file:
+
+    ```yaml
+    # datadog-agent.yaml file
+    spec:
+      features:
+        remoteConfiguration:
+          enabled: true
+        cws:
+          enabled: true
+    ```
+
+2. Restart the Agent.
```

Comment:
I think technically it's more correct to say `Apply the changes` then restart the agent in the case of k8s Operator installation